EIPS/eip-2515.md

```diff
@@ -1,6 +1,6 @@
 ---
-eip: 2512
```

Comment:
How was this merged with the wrong number in the first place?

Reply:
No idea. It confuses me.